### ISSUE TITLE
Firefox Image Map Click Through - Possible Fix

### ISSUE THREAD

**Issue by marketinview** [thdoan/magnify#59]
Currently, the [image map demo](https://thdoan.github.io/magnify/demo-map.html) click through doesn't work in Firefox..it requires two clicks. I had the need to do a click through and found a method that works in Firefox (as well as Chrome and Edge) to fire a click at a specific x,y:

```
function clickxy(x, y) {
    var ev = new MouseEvent('click', {
        'view': window,
        'bubbles': true,
        'cancelable': true,
        'clientX': x,
        'clientY': y
    });
    var el = document.elementFromPoint(x, y);
    el.dispatchEvent(ev);
} 
```

**Comment by thdoan** [comment]
This has a variable scope issue and has been fixed in v2.3.3. Please test the update demo you linked to above (Shift + F5) and close if it works for you.

**Comment by marketinview** [comment]
Fix confirmed.
